setup.py

```diff
@@ -37,5 +44,5 @@
        package_data         = {'': ['*.ui']},
        license              = 'GPL',
        zip_safe             = False,
-       install_requires     = [ 'scapy', 'dpkt', 'NetfilterQueue', 'psutil' , 'pyinotify']
+       install_requires     = [ 'scapy', 'dpkt', 'NetfilterQueue', 'psutil' , 'pyinotify', 'PyQt5']
```

Comment:
"setup.py install" fails to install scapy with python 3, `'scapy-python3'` worked for me